src/cryptonote_protocol/cryptonote_protocol_handler.inl

```diff
@@ -307,7 +307,7 @@ namespace cryptonote
     uint64_t diff_v2 = max_block_height > last_block_v1 ? std::min(abs_diff, max_block_height - last_block_v1) : 0;
```

Comment:
I had a look and `diff_v2` looks like bogus code that isn't relevant to us and some of the other lines as well. These lines, 305-308 look to be useless and actually leaving it in the code is a time ticking bomb waiting to crash on the 1 millionth block.

```
uint64_t max_block_height = std::max(hshd.current_height,m_core.get_current_blockchain_height());
uint64_t last_block_v1 = m_core.get_nettype() == TESTNET ? 624633 : m_core.get_nettype() == MAINNET ? 1009826 : (uint64_t)-1;
uint64_t diff_v2 = max_block_height > last_block_v1 ? std::min(abs_diff, max_block_height - last_block_v1) : 0;
assert(diff_v2 == 0);
```
So I'd remove them in this commit as well.